src/solvers/dg_p4est/containers.jl

```diff
@@ -100,13 +100,51 @@ end
 @inline nboundaries(boundaries::BoundaryContainerP4est) = length(boundaries.element_ids)
 
 
+struct MortarContainerP4est{NDIMS, uEltype<:Real, NDIMSP1, NDIMSP3} <: AbstractContainer
+  u::Array{uEltype, NDIMSP3}       # [small/large side, variable, position, i, j, mortar]
+  element_ids::Matrix{Int}         # [position, mortar]
+  node_indices::Matrix{NTuple{NDIMS, Symbol}} # [small/large, mortar]
```

Comment:
I usually prefer writing `Matrix{T}` whenever I want to have a linear operator and `Array{T, 2}` whenever I just have some array that happens to have two indices. That clarifies the purpose for me.